src/Compilers/VisualBasic/Portable/Binding/MemberSemanticModel.vb

```diff
@@ -281,6 +281,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                      SyntaxKind.ExitWhileStatement,
                      SyntaxKind.ExitFunctionStatement,
                      SyntaxKind.ExitSubStatement,
+                     SyntaxKind.ExitOperatorStatement,
+                     SyntaxKind.ExitPropertyStatement,
```

Comment:
Doc comment for this function states that it is supposed to be called only for statements that can be in executable code. Why are we calling it for Exit Property/ Exit Operator and the like?